extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -148,12 +169,25 @@
 
                 // Set protocol
                 GuacamoleConfiguration config = new GuacamoleConfiguration();
+                
+                GuacamoleProxyConfiguration proxyConfig;
+                try {
+                    proxyConfig = new LocalEnvironment().getDefaultGuacamoleProxyConfiguration();
+                }
+                catch (GuacamoleException e) {
+                    return null;
```

Comment:
If retrieval of the proxy config fails, that failure should be logged rather than silently ignored.

Reply:
Got it.

Reply:
Logged.